.pre-commit-config.yaml

```diff
@@ -31,12 +31,9 @@ repos:
               )$
         -   id: check-added-large-files
         -   id: check-executables-have-shebangs
+            files: ^Tools/
```

Comment:
why restrict only on Tools ? we should fix permissions on all files and we got py script into libraries/ too (specially on AP_HAL_Chibios)

Reply:
https://github.com/ArduPilot/ardupilot/pull/30288#discussion_r2136752562